app/src/main/java/com/baosystems/icrc/psm/di/ServicesModule.kt

```diff
@@ -1,10 +1,10 @@
 package com.baosystems.icrc.psm.di
 
-import com.baosystems.icrc.psm.services.*
-import com.baosystems.icrc.psm.services.rules.RuleValidationHelper
-import com.baosystems.icrc.psm.services.rules.RuleValidationHelperImpl
-import com.baosystems.icrc.psm.services.scheduler.BaseSchedulerProvider
-import com.baosystems.icrc.psm.services.scheduler.SchedulerProviderImpl
+import org.dhis2.android.rtsm.services.*
```

Comment:
Wildcards imports are not allowed by ktlint. It may throw a warning/error message when checked.